src/sessionManager.ts

```diff
@@ -144,33 +148,60 @@ export default function SessionManager(
             return;
         }
 
+        // debugger;
+
         let sessionTimeoutInMilliseconds: number;
         let timeSinceLastEventSent: number;
 
-        const cookies: IPersistenceMinified = mpInstance._Persistence.getPersistence();
+        const persistence: IPersistenceMinified = mpInstance._Persistence.getPersistence();
 
-        // TODO: https://go.mparticle.com/work/SQDSDKS-5684
-        if (!cookies) {
-            return;
-        }
+        // debugger;
 
-        if (cookies.gs && !cookies.gs.sid) {
+        // QUESTION: What constitutes ending a session if we don't have persistence?
```

Comment:
Addressed in earlier comments, but if we don' have persistence, then the expectation is that everything is saved to the store, so the logic would be if persistence is disabled, check the store instead